var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -50,11 +50,12 @@ class Adios(AutotoolsPackage):
 
     variant('mpi', default=True, description='Enable MPI support')
     variant('infiniband', default=False, description='Enable infiniband support')
+    variant('mxml', default=False, description='Use external MXML library')
```

Comment:
I wouldn't actually care, if the mxml configure and build were not broken. It is extremely tiresome to fix N different dependent packages just to get the 1 package actually built!